portfolio/src/main/webapp/script.js

```diff
@@ -85,10 +88,10 @@ function createLinkElement(link, text) {
   return linkElement;
 }
 
-/** Creates an <li> element containing text. */
-function createListElement(text) {
+/** Creates an <li> element containing the comment and the user's email. */
+function createListElement(text, email) {
   const liElement = document.createElement('li');
-  liElement.innerText = text;
+  liElement.innerText = '"' + text + '" [by ' + email + ']';
```

Comment:
Check if email is null or undefined, and maybe show "Anonymous" or something if that's the case.

Reply:
It's not possible to see the comments form if you're not logged in, so it will always collect a user email.